modules/atlas/splitmerge/meta.yml

```
@@ -0,0 +1,73 @@
+name: "atlas_splitmerge"
+description: split single end read groups by length and merge paired end reads
+keywords:
+  - split
+  - merge
+  - bam
+  - read group
+tools:
+  - "atlas":
+      description: "ATLAS, a suite of methods to accurately genotype and estimate genetic diversity"
+      homepage: "https://bitbucket.org/wegmannlab/atlas/wiki/Home"
+      documentation: "https://bitbucket.org/wegmannlab/atlas/wiki/Home"
+      tool_dev_url: "https://bitbucket.org/wegmannlab/atlas"
+      doi: "10.1101/105346"
+      licence: "['GPL v3']"
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test', single_end:false ]
+
+  - bam:
+      type: file
+      description: The one and only BAM file.
+      pattern: "*.{bam}"
```

```suggestion
      pattern: "*.bam"
```

Curly brackets not necessary if not a list (same goes for everything else)